scipy/stats/_continuous_distns.py

```diff
@@ -3779,6 +3778,12 @@ class rice_gen(rv_continuous):
 
     `rice` takes ``b`` as a shape parameter.
 
+    The Rice distribution describes the length, `r`, of a 2-D vector
+    with components `(U+u, V+v)`, where `U, V` are constant, and `u, v`
+    are independent Gaussian random variables with standard deviation
+    `s`.  Let `R = (U**2 + V**2)**0.5`. Then the pdf of `r` is
+    `rice.pdf(x, R/s, scale=s)`.
```

Comment:
Typesetting nitpick: I think this needs double backticks for ReST layout (double backticks for code, single backticks for cross-references). Or explicit `:math:` blocks.